### DISCUSSION TITLE
Connection problems

### DISCUSSION THREAD

**Discussion by serhat-m** [gysmo38/mitsubishi2MQTT#235] [General]
Hello everyone, I have the following problem:

when I try to connect the device to my network, the device reboots and the HVAC_XXXX network is displayed again. Out of 10 attempts it occasionally connects but then as soon as you make a change in the UI and reboot, it becomes visible as HVAC_XXXX after the reboot. Could not detect any unusual settings on my router. Any idea?

Setup:
Controller: ESP8266 Adafruit HUZZAH
Router: ASUS-AX88U, 2,4GHz network for the ESP without Channel 12,13 and a simple password

**Comment by Noori1972** [comment]
It sounds like you're encountering connectivity issues with your ESP8266 device when trying to connect it to your ASUS-AX88U router's 2.4GHz network. The issue might be related to various factors, such as signal interference, DHCP conflicts, or firmware glitches. Here are some steps you can take to troubleshoot and potentially resolve the problem:

Firmware Update: Ensure that your ESP8266's firmware is up to date. Sometimes, outdated firmware can cause connectivity issues. Check if there are any updates available for your device.

Wi-Fi Signal Strength: Make sure that the ESP8266 is within a reasonable range of your router to ensure a strong and stable Wi-Fi signal. Walls and other obstacles can weaken the signal.

Signal Interference: Check if there are any other electronic devices, appliances, or neighboring Wi-Fi networks operating on the same frequency (2.4GHz) that might be causing interference. Microwave ovens, cordless phones, and other Wi-Fi networks can potentially disrupt the connection.

Channel Selection: Although you mentioned that you're not using Channels 12 and 13, it's still worth experimenting with different Wi-Fi channels to see if you get better stability. Some devices might have issues with certain channels due to local regulations and restrictions.

Static IP Assignment: Instead of relying on DHCP, try assigning a static IP address to your ESP8266 device. This can help prevent any potential IP conflicts and ensure a consistent connection.

Network Settings: Double-check your router's network settings to ensure that there are no special configurations or settings that might be causing conflicts with the ESP8266. Make sure MAC address filtering or any security features aren't inadvertently causing issues.

Power Supply: Ensure that your ESP8266 is receiving a stable power supply. Insufficient power can lead to unexpected reboots and connectivity issues.

Reset to Factory Defaults: If you've made a lot of changes to the ESP8266's settings, consider resetting it to factory defaults and starting the setup process again. Sometimes, incorrect configurations can cause issues.

Debugging: If possible, try to access debugging logs or error messages from the ESP8266. This might give you more insight into what's causing the device to reboot and lose connection.

Check for Known Issues: Look for any known issues related to your specific ESP8266 model and the router. Sometimes, there might be compatibility or firmware-related problems that others have already identified and addressed.

Contact Manufacturer/Community: If you've tried all the above steps and the issue persists, it might be a good idea to reach out to the manufacturer's support or relevant online communities where others might have faced and resolved similar issues.

Remember to document the changes you make during troubleshooting, so you can easily revert to previous settings if needed.
